test/javascripts/controllers/reorder-categories-test.js.es6

```diff
@@ -30,7 +30,7 @@ QUnit.test("fixIndices set unique position number", function(assert) {
 });
 
 QUnit.test(
-  "fixIndices places subcategories after their parent categories, while maintaining the relative order",
+  "reorder places subcategories after their parent categories, while maintaining the relative order",
```

Comment:
We probably want to add another test that ensure sub-sub-categories work properly?

Reply:
Amended the current test to include a sub-sub-category.